mobile/src/main/java/org/openhab/habdroid/model/LinkedPage.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ *   All rights reserved. This program and the accompanying materials
+ *   are made available under the terms of the Eclipse Public License v1.0
+ *   which accompanies this distribution, and is available at
+ *   http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.habdroid.model
+
+import android.os.Parcelable
+
+import kotlinx.android.parcel.Parcelize
+import org.json.JSONObject
+import org.openhab.habdroid.util.forEach
+import org.w3c.dom.Node
+
+/**
+ * This is a class to hold information about openHAB linked page.
+ */
+
+@Parcelize
+data class LinkedPage(val id: String?, val title: String?, val icon: String?,
+                      val iconPath: String, val link: String) : Parcelable {
```

Comment:
Are `id` and `title` really nullable (IOW, optional)? It's been like that since [like forever](https://github.com/openhab/openhab-android/blob/7aa2593b9e6f41d7c479b4c29879b7ede0fcf801/mobile/src/main/java/org/openhab/habdroid/model/OpenHABLinkedPage.java#L52-L59), but it doesn't look correct to me at least for for id on OH2.

Reply:
None of these make sense to be nullable. I suggest we make it non-nullable and look for crash reports.